config/sync/core.menu.static_menu_link_overrides.yml

```diff
@@ -0,0 +1,19 @@
+definitions:
```

Comment:
@pookmish Do we need this file? What do you think about me getting rid of it?

Reply:
it is part of core config. i'd be nervous about deleting it. https://git.drupalcode.org/project/drupal/-/tree/9.0.x/core/config/install